test/net/sourceforge/kolmafia/request/PlaceRequestTest.java

```diff
@@ -74,7 +77,56 @@ void itShouldSetTheToolbeltAsAFreePullInTTT() {
     assertEquals(
         -1, freePulls.indexOf(toolbelt), "toolbelt should not be in Free Pulls after TTT fades");
   }
-}
 
-// Generated with love by TestMe :) Please report issues and submit feature requests at:
-// http://weirddev.com/forum#!/testme
+  @Test
```

Comment:
Why did you un-Nest these tests?
It was much cleaner to have them in a Nested class just for whichplace=speakeasy